utils/build/run-driver-posix.sh

```diff
@@ -1,3 +1,30 @@
-#!/bin/sh
```

Comment:
We are going from 3 line script to 30 line script for a platform we don't really support. We have no idea if this fails the moment you start browsers on nix, etc. So I would be reluctant to landing something like this.

Reply:
Hello @pavelfeldman :wave: 

> We are going from 3 line script to 30 line script for a platform we don't really support. 

Yea that is a fair argument.

> We have no idea if this fails the moment you start browsers on nix, etc. So I would be reluctant to landing something like this.

OK; fair enough it's "your" project if you say you can not test or support this code I will accept that and have to use my monkey patched version.